components/scream/src/mct_coupling/scream_cpl_indices.F90

```diff
@@ -117,6 +125,11 @@ subroutine scream_set_cpl_indices (x2a, a2x)
     scr_names_x2a(20) = 'unused'
     scr_names_x2a(21) = 'unused'
 
+    ! No scream imports currently require vector component info. Set to -1.
```

Comment:
Side question: I saw `surf_u_mom_flux` and `surf_v_mom_flux`, and thought "why, aren't those two components of a vector?" So I grepped for them, and found that they are two distinct fields in SHOC. Since you are a shoc guru too, do you think they can be fused into a vector field in SHOC?

Reply:
Absolutely.